client/my-sites/plans/current-plan/current-plan-thank-you/anti-spam-thank-you.tsx

```diff
@@ -19,22 +22,41 @@ const ThankYouCta: ThankYouCtaType = ( { dismissUrl, recordThankYouClick } ) =>
 	);
 };
 
-const AntiSpamProductThankYou = (): ReactElement => {
+const AntiSpamProductThankYou = ( { installProgress } ): ReactElement => {
 	const translate = useTranslate();
+	const isInstalled = installProgress === 100;
+
 	return (
 		<ThankYou
 			illustration="/calypso/images/illustrations/thankYou.svg"
 			title={ translate( 'Say goodbye to spam!' ) }
-			ThankYouCtaComponent={ ThankYouCta }
+			showHideMessage={ ! isInstalled }
+			ThankYouCtaComponent={ isInstalled && ThankYouCta }
 		>
-			<p>{ translate( "We're setting up Jetpack Anti-Spam for you right now." ) }</p>
-			<p>
-				{ translate(
-					"In no time you'll be able to enjoy more peace of mind and provide a better experience to your visitors."
+			<>
+				<p>{ translate( "We're setting up Jetpack Anti-Spam for you right now." ) }</p>
+				<p>
+					{ translate(
+						"In no time you'll be able to enjoy more peace of mind and provide a better experience to your visitors."
+					) }
+				</p>
+				{ ! isInstalled && (
+					<ProgressBar
+						isPulsing={ ! isInstalled }
```

Comment:
Wouldn't this always equal true given this component not show if `isInstalled` is true?

Reply:
yes! I forgot to restore it after talking to Filipe, I was trying with another design. Thanks :)